### PR TITLE
Fix getSourcefile in BaseFile class

### PR DESCRIPTION
**Pull request by dankwart-de**
A false parameter was passed within the method and this lead to a
ClassCastException

### CODE REVIEW

File: dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflow.java [hunk 1]
```diff
@@ -17,6 +18,7 @@
     @Override
     public boolean execute(ExecutionContext context) {
         SimpleRuntimeService srs = (SimpleRuntimeService) context.getRuntimeService();
+        BaseFile abc = getSourceFile(context, "/tmp/test.txt");
```
Comment (vinjana) [inline dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflow.java]: I am not a fan of polluting /tmp. Roddy already puts so many files there that have to be cleaned up manually. We should consider making an issue for cleaning up /tmp after tests.

Here the file should be created in a dedicated subdirectory for /tmp.

BTW: `abc`?

File: dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflowWithCorrectExecution.java [hunk 2]
```diff
@@ -17,6 +18,7 @@
     @Override
     public boolean execute(ExecutionContext context) {
         SimpleRuntimeService srs = (SimpleRuntimeService) context.getRuntimeService();
+        BaseFile abc = getSourceFile(context, "/tmp/test.txt");
```
Comment (vinjana) [inline dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflowWithCorrectExecution.java]: dito

File: dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflow.java (hunk 1 above)
Comment (dankwart-de) [reply dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflow.java]: This is really only a test workflow which is currently not used in any kind of automatic test.
The file also does not exist.

File: dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflowWithCorrectExecution.java (hunk 2 above)
Comment (dankwart-de) [reply dist/plugins/TestPluginWithJarFile/src/de/dkfz/roddy/knowledge/examples/SimpleWorkflowWithCorrectExecution.java]: dito

**Pull request merged**